bundles/io/org.eclipse.smarthome.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/discovery/DiscoveryResource.java

```diff
@@ -91,7 +101,9 @@ public void onFinished() {
             }
         });
 
-        return Response.ok(discoveryServiceRegistry.getMaxScanTimeout(bindingId)).build();
+        DiscoveryServiceDTO dto = new DiscoveryServiceDTO();
+        dto.timeout = discoveryServiceRegistry.getMaxScanTimeout(bindingId);
```

Comment:
I would like to avoid just returning a single value that is not future proof. At the moment, we’re just returning the time, but what if in the future we want to also return some other information from the discovery? I don’t currently have anything in mind right now, but if we start with returning just the timeout without the possibility of returning additional information, then it will not be easily possible to change it later.